moja-kuhinjica-frontend/src/components/modal/passwordForgetting/PasswordForgettingModal.tsx

```diff
@@ -0,0 +1,92 @@
+import React, { useState } from 'react'
+import Modal from 'react-modal'
```

Comment:
Add forgot and reseting modal mobile version

Reply:
Done.